packages/core/helper-plugin/package.json

```diff
@@ -90,6 +90,6 @@
     }
   },
   "peerDependencies": {
-    "webpack": "^5.74.0"
+    "webpack": "^5.75.0"
```

Comment:
I'm not sure webpack is a peer dep of helper-plugin because it's shipped pre-bundled?

Reply:
Oh it is - this PR just updates the version. If we want to change that we can do it in a follow-up PR. I think your bundling PR already takes care of that?